pennylane/transforms/adjoint.py

```diff
@@ -18,34 +18,51 @@
 
 
 def adjoint(fn):
-    """Create a function that applies the adjoint of the provided operation or template.
+    """Create a function that applies the adjoint (inverse) of the provided operation or template.

```

Comment:
@Thenerdstation I updated this docstring after discussion with a user that was having trouble